src/datastore/db.ts

```diff
@@ -0,0 +1,128 @@
+import fs from 'fs/promises';
+import path from 'path';
+
+interface Index {
+  [key: string]: string[];
+}
+
+const ERROR_NOT_OPENED = 'Db wasn`t opened';
+const ERROR_NO_SUCH_MODEL = 'no such model';
+const ERROR_NO_SUCH_RECORD = 'no such record';
+
+export class DB {
+  index: Index | null = null;
+  path: string;
```

Comment:
Change access modifiers 
```suggestion
  private index: Index | null = null;
  private path: string;
```